tests/common/plugins/custom_skipif/__init__.py

```diff
@@ -0,0 +1,217 @@
+import multiprocessing
+import yaml
+import pytest
+import logging
+import os
+import sys
+import subprocess
+
+
+logger = logging.getLogger()
+
+
+def pytest_collection(session):
+    initialize_cached_variables(session)
+
+
+def initialize_cached_variables(session):
+    custom_skip_if_dict = 'custom_skip_if_dict'
```

Comment:
My general inclination is to put constants like this at the top of the file in all caps i.e.

```
CUSTOM_SKIP_IF_DICT = "custom_skip_if_dict"
```

this way if you have multiple references throughout the file you can reference a single variable at the top of the file. The all caps is just code style.

Reply:
will fix